chapter05/drjt/ex1

```diff
@@ -0,0 +1 @@
+The two hash assigments are recieving anonymous hash refs, when its expecting an even-numbered list
```

Comment:
Yep, pretty much.  The curly brackets denote a scalar - while the %foo assignment requires an even list.

Also, Ginger is a big fat liar.